contract/src/internal.rs

```diff
@@ -121,10 +121,8 @@ impl NearCLP {
         let mut p = self.must_get_pool(&token);
         let near_to_pay = self.calc_in_amount(tokens_out, p.ynear, p.reserve);
         // panics if near_to_pay > max_near_paid
```

Comment:
next line underflows if near_to_pay > max_near_paid

Reply:
in fact it was desired (as the comment said) :)